src/algebra/gcd_monoid.lean

```diff
@@ -376,13 +413,32 @@ lcm_dvd_lcm (dvd_refl _) (dvd_mul_right _ _)
 
 end lcm
 
-end gcd_domain
+namespace gcd_monoid
+theorem prime_of_irreducible {x : α} (hi: irreducible x) : prime x :=
+begin
+  split, apply hi.ne_zero, --unfold irreducible at hi,
```

Comment:
Do you want to keep this comment?

Reply:
Also, after `split`, I would expect to see `{ }` around the separate subproofs. Same for the next line.

Reply:
I've wrapped the main proof in a bit of term mode instead -- does it look reasonable?